### DISCUSSION TITLE
Habitat builder deconstruct controls?

### DISCUSSION THREAD

**Discussion by Telkin1** [Okabintaro/SubmersedVR#52] [Q&A] [answered]
Hi, I've recently started playing Subnautica with SubmersedVR (huge thanks to everyone who worked on this mod!)

My question is: how do I deconstruct my base with the habitat builder, which button do I have to press? I'm using the Oculus Quest 1 controllers.

**Comment by Okabintaro** [comment, marked as answer]
Hello, to deconstruct on the quest you need to pull the left thumb stick down. To see or modify all the bindings and controls you can open the bindings on the SteamVR dashboard.

**Reply by Telkin1** [reply]
Thanks for the quick answer! I didn't know that about the the SteamVR dashboard.
